config.yaml

```diff
@@ -72,6 +72,11 @@ options:
       The public-facing base URL that clients use to access this Homeserver.
       Defaults to https://<server_name>/. Only used if there is integration with
       SAML integrator charm.
+  ready_check:
+    type: string
+    description: Comma separated list of period,threshold and timeout for Synapse
+      pebble ready check.
+    default: "2m,5,20s"
```

Comment:
Just out of curiosity, what are the time units that can be specified (s/m/h)? I'm wondering if it might help to add them to the description as well :D

Reply:
It's not specified but it should follow the same rules defined by pebble layer. I'll add a link
https://github.com/canonical/pebble/#layer-specification